conventions/documentation/actionmenus.md

```diff
@@ -4,24 +4,75 @@ parent: Conventions
 
 # Action menus
 
-## Kebab and resource ‘Actions’ menus
+## Kebab and resource Actions menus
 
-- List views of resources have a “kebab” menu that matches the resources’ “Actions” menu in their details view.
+- A resource's Actions menu is accessible from a number of places, such as:
+  1. List view kebab
+  2. Details page actions dropdown
+  3. Topology side panel
+  4. Topology right-click action on a resource
+- By default, submenus will open to the right of the main menu. In cases where there is no space to the right, they can open to the left.
 
+**List view kebab**
 ![action menus kebab](../images/action-menus-kebab.png)
 
+**Details page actions dropdown**
 ![action menus actions](../images/action-menus-actions.png)
 
-Ordering of actions in the menu should follow the following pattern:
-- Unique actions to that resource should appear above default resource actions, sorted by most frequently used first.
-- Edit Labels (when present)
-- Edit Annotations (when present)
-- Edit [Resource Name]
-- Delete [Resource Name]
+**Topology side panel**
+![action menus topology panel](../images/action-menus-topology-panel.png)
+
+**Topology right click action on a resource**
+![action menus topology](../images/action-menus-topology.png)
+
+- Actions dropdowns should be styled as primary blue dropdowns.
+- Exceptions are acceptable in cases where a different action(s) has been identified as the view's primary action. In these cases, the primary action(s) is usually pulled out of the dropdown and is styled as a primary button or dropdown to the left of the Actions dropdown, which will thus have secondary styling (e.g., Secrets details pages).
+
+## Organization of Actions menu items
+
+**Basic action menus**
+
+<img src="../images/action-menus-1.png" alt="Action menus order" width="294"/>
+
+The majority of resource action menus will be a basic action menu. These menus have "universal" actions and the `Delete {resource_type}` action separated by a horizontal divider.
+
+**Complex action menus**
+
+Some resource action menus will have more actions than those included in the basic action menu. The following documentation uses a Deployment as an example, but the logic to each section should be applied to any other complex action menu.
+
+<img src="../images/action-menus-2.png" alt="Action menus order" width="1239"/>
+
+Action menus should be separated into multiple sections. The ordering of sections and actions in the menu should adhere to the following pattern:
+1. **Resource specific actions**
+  * These should be ordered with standalone actions listed first followed by flyouts, with each separate group ordered alphabetically by default.
```

Comment:
I was wondering if it was worth trying to describe how to order the standalone actions, only because I think that had to be added to the Actions convention at some point. At the time I believe the thought was "sorted by most frequently used first" but I don't know if we still want to follow that. Maybe it would be fine to do alphabetical now since there might generally be less standalone actions since many fall into the flyouts.